### PR TITLE
refactor: [TD-1668] Update `orderbook` OpenAPI SDK

### PR DESCRIPTION
**Pull request by lfportal**
### Hi👋, please prefix this PR's title with:

- [ ] `breaking-change:` if you have introduced modification that necessitates immediate adjustments by this SDK's users to their applications, clients, or integrations to avert disruptions to existing features or functionalities.
- [x] `feat:`, `fix:`, `refactor:`, `docs:`, or `chore:`.

# Summary

Update internal `orderbook` OpenAPI SDK.

# Detail and impact of the change

## Changed

* The orderbook package's `Order` type has been modified into a union of multiple order types in preparation for introducing bids.

### CODE REVIEW

**Review by naveen-imtb — APPROVED** [review]

**Review by naveen-imtb — APPROVED** [review]

**Review by naveen-imtb — DISMISSED** [review]

**Review by Sam-Jeston — DISMISSED** [review]

**Review by Sam-Jeston — DISMISSED** [review]

File: packages/orderbook/src/orderbook.ts [hunk 1]
```diff
@@ -114,21 +116,21 @@ export class Orderbook {
   }
 
   /**
-   * Get an order by ID
+   * Get an listing by ID
```
Comment (Sam-Jeston) [inline packages/orderbook/src/orderbook.ts:119]:
```suggestion
   * Get a listing by ID
```

File: sdk/package.json [hunk 2]
```diff
@@ -8,6 +8,7 @@
   "dependencies": {
     "@0xsequence/abi": "^1.4.3",
     "@0xsequence/core": "^1.4.3",
+    "@0xsquid/sdk": "^2.8.24",
```
Comment (Sam-Jeston) [inline sdk/package.json:11]: not sure how this diff ended up here. Maybe a weird merge with main? Its tagged the SDK team for review because of it I think

**Review by Sam-Jeston — APPROVED** [review]
Some very minor things, but looks great. Very clean

File: sdk/package.json (hunk 2 above)
Comment (lfportal) [reply sdk/package.json:11]: After running through the steps to rebuild the SDK, these things popped in 🤷

**Review by naveen-imtb — DISMISSED** [review]

**Review by naveen-imtb — APPROVED** [review]

**Review by zaidarain1 — APPROVED** [review]

**Pull request merged**